openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -587,107 +591,110 @@ def run(self, *, dry=False, verbose=True,
             capacity=None, time_to_live=None, platform=platform,
         )
 
-        # 11. Set the integrator
-        #  a. get integrator settings
-        integrator_settings = prototol_settings.integrator_settings
-
-        #  b. create langevin integrator
-        integrator = openmmtools.mcmc.LangevinSplittingDynamicsMove(
-            timestep=to_openmm(integrator_settings.timestep),
-            collision_rate=to_openmm(integrator_settings.collision_rate),
-            n_steps=integrator_settings.n_steps.m,
-            reassign_velocities=integrator_settings.reassign_velocities,
-            n_restart_attempts=integrator_settings.n_restart_attempts,
-            constraint_tolerance=integrator_settings.constraint_tolerance,
-            splitting=integrator_settings.splitting
-        )
-
-        # 12. Create sampler
-        if sampler_settings.sampler_method.lower() == "repex":
-            sampler = _rfe_utils.multistate.HybridRepexSampler(
-                mcmc_moves=integrator,
-                hybrid_factory=hybrid_factory,
-                online_analysis_interval=sampler_settings.online_analysis_interval,
-                online_analysis_target_error=sampler_settings.online_analysis_target_error.m,
-                online_analysis_minimum_iterations=sampler_settings.online_analysis_minimum_iterations
-            )
-        elif sampler_settings.sampler_method.lower() == "sams":
-            sampler = _rfe_utils.multistate.HybridSAMSSampler(
-                mcmc_moves=integrator,
-                hybrid_factory=hybrid_factory,
-                online_analysis_interval=sampler_settings.online_analysis_interval,
-                online_analysis_minimum_iterations=sampler_settings.online_analysis_minimum_iterations,
-                flatness_criteria=sampler_settings.flatness_criteria,
-                gamma0=sampler_settings.gamma0,
+        try:
```

Comment:
Does the try really have to be all the way up at the integrator generation stage? We shouldn't be failing here ever. Why not just over the if/else on dry run?